Python job fails on nightly: Missing `setuptools`

Nightly is failing. [setuptools is missing](https://github.com/opendp/opendp/actions/runs/8369403837/job/22915415668#step:6:38):
```
Build bdist
$ (cd python && python setup.py bdist_wheel -d wheelhouse)
Traceback (most recent call last):
  File "/home/runner/work/opendp/opendp/python/setup.py", line 1, in <module>
    from setuptools import setup  # type: ignore
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'setuptools'
Error: Process completed with exit code 1.
```

Easy fix?

Also, consider adding a nightly status badge on the README so this will be a higher priority for us going forward.

--- a/tools/python_build.sh
+++ b/tools/python_build.sh
@@ -38,7 +38,7 @@ function run() {
 
 function init() {
   log "Install dependencies"
-  run pip install wheel setuptools
+  run pip install setuptools wheel
 }
 
 function build() {

--- a/tools/python_build.sh
+++ b/tools/python_build.sh
@@ -38,7 +38,7 @@ function run() {
 
 function init() {
   log "Install dependencies"
-  run pip install wheel
+  run pip install wheel setuptools
 }
 
 function build() {

